apps/mobile/src/components/InsideRouter/components/SettingsScreen/components/ButtonsSection.tsx

```diff
@@ -36,11 +36,36 @@ import openUrl from '../../../../../utils/openUrl'
 import ChangeCurrency from './ChangeCurrency'
 import {type Currency} from '@vexl-next/domain/src/general/offers'
 import {selectedCurrencyAtom} from '../../../../../state/selectedCurrency'
+import {importedContactsCountAtom} from '../../../../../state/contacts'
 
 const ItemText = styled(Text, {
   fos: 18,
 })
 
+const SubtitleText = styled(Text, {
+  fos: 12,
+  col: '$greyOnBlack',
+})
+
+function ContactsImportedTitle({
```

Comment:
Would it be possible to move these components to another file? This file is already super long. It might improve code readability.